content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Istio Webhook Management
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 10
+keywords: [security,webhook]
+---
+
+{{< warning >}}
```

Comment:
Use the boilerplate for this warning please.

Reply:
Done.